mongodbatlas_test.go

```diff
@@ -190,6 +190,54 @@ func TestAcceptanceDatabaseUser_CreateUser(t *testing.T) {
 	assertCredsExists(t, projectID, publicKey, privateKey, createResp.Username, password, connURL, testMongoDBAtlasRole)
 }
 
+func TestAcceptanceDatabaseUser_CreateUserWithTemplate(t *testing.T) {
```

Comment:
This looks great! Can you also add a check (either in an existing test or a new one) that checks the default username template?

Reply:
Thanks! Added a new test for the default template.